### ISSUE TITLE
MameHub won't load games

### ISSUE THREAD

**Issue by pedetentous** [MisterTea/MAMEHub#354]
Hello! So, I downloaded the automated MAMEHub 3.1.0 installer from the website (http://www.mamehub.info/download). It managed to load it fine on my MacBook Pro, but I can't actually play any games. I start a game and it looks like it immediately stops.

This is what happens in Terminal when I double click a game...

>[04:20:19,853  INFO MainFrame.java:608] Got system Arcade and rom RomInfo(id:carpolo, cloneRom:null, parentRom:null, filenames:[/Users/Ped/Downloads/MAMEHub/MAMEHub/MAMEHubRepo/Binaries/dist/../roms/carpolo.zip], description:Car Polo, missingReason:null, softwareLists:[], system:Arcade, interfaceFileMap:{})(0)
>[04:20:19,958  INFO RpcEngine.java:366] GOT MYSELF: Player(id:I_pedetentous, name:pedetentous, ipAddress:108.74.30.131, loggedIn:true, moderator:false, inGame:, portsOpen:true, status:PlayerStatus(chatStatus:ONLINE, operatingSystem:MAC), basePort:6805, secondaryPort:6806)
>[04:20:20,153  INFO MameHubEngine.java:143] CALLING: ./csume64 "/Users/Ped/Downloads/MAMEHub/MAMEHub/MAMEHubRepo/Binaries/dist/../roms/carpolo.zip" -port 6805 -server -username pedetentous 
>[04:20:20,321  INFO MainFrame.java:1652] UPDATED WITH 1 (1) VALUES
>[04:20:20,321  INFO MainFrame.java:1572] PLAYER CHANGED
>[04:20:20,321  INFO MainFrame.java:1608] GAME CHANGED: Game(id:2kby8nzzf6ry56go, system:Arcade, rom:carpolo, locked:false, startTime:1441527619987, endTime:0, hostPlayerId:I_pedetentous, hostPlayerIpAddress:108.74.30.131, hostPlayerPort:6805)
>[04:20:20,322  INFO MainFrame.java:1652] UPDATED WITH 2 (2) VALUES
>[04:20:21,184  INFO MameHubEngine.java:192] RESULT: 133
>[04:20:21,184  INFO MainFrame.java:1680] LEAVING GAME
>[04:20:21,571  INFO MainFrame.java:1608] GAME CHANGED: Game(id:2kby8nzzf6ry56go, system:Arcade, rom:carpolo, locked:false, startTime:1441527619987, endTime:1441527621215, hostPlayerId:I_pedetentous, hostPlayerIpAddress:108.74.30.131, hostPlayerPort:6805)

I figure that the "RESULT:133" is the issue?
After doing a little digging of my own, I found something someone else tried. Loading the game directly using the following code and its variants...

>./csume64 "/Users/Ped/Downloads/MAMEHub/MAMEHub/MAMEHubRepo/Binaries/dist/../roms/carpolo.zip" -port 6805 -server -username pedetentous

I keep getting the following error...

>dyld: Library not loaded: @rpath/libclang_rt.asan_osx_dynamic.dylib
>Referenced from: /Users/Ped/Downloads/MAMEHub/MAMEHub/MAMEHubRepo/Binaries/dist/./csume64
>Reason: image not found
>Trace/BPT trap: 5

Am I doing something wrong? There were no other instructions except to run the sh script. I did also install the Java JDK on my own to get this working as well.

**Comment by pedetentous** [comment]
Okay, I managed to get this working by downloading  the 3.7.0 version of "Clang for Mac OS X" from LLVM.org directly. I renamed the unzipped folder to "llvm370" and placed it in a "Builds" folder in my User folder. I then modified my .bash_profile to link to the clang within that folder using:

`export DYLD_LIBRARY_PATH=~/Builds/llvm370/lib/clang/3.7.0/lib/darwin`

The game loads, but now I get two small blue boxes near the top left and right corners of the game. Is there something that can fix that?
